.travis.yml

```diff
@@ -1,12 +0,0 @@
-language: node_js
-node_js:
-  - 12
-  - 10
-  - 8
-install:
-  - "pip install --user -U pip"
-  - "pip install --user git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic"
-before_script:
-  - "! grep -Fq '.only(' test/**/*.spec.js"
```

Comment:
Dropping this line is dangerous - if anyone accidentally commits a `.only` then only that unit test will run. This happens more than you'd think.

It _may_ be covered by our eslint rules.

It'd be worth making sure it's caught somewhere. If it's really hard to implement in GA then raise an issue for it to unblock this PR.

Reply:
I'm not setting the  `--forbid-ony` flag when running mocha. If there are any `.only` it will fail.